src/assets/partials/_contact.scss

```diff
@@ -0,0 +1,88 @@
+.contact-title {
+	color: #FFFFFF; //TODO variables
+	text-align: center;
+	margin: 2% 0 8% 0;
+	text-transform: uppercase;
+	font-size: 16px;
+}
+
+.link-contact[type=checkbox] {
+	visibility: hidden;
+}
+
+.link-contact[type=checkbox]:checked ~ .contact-form {
+	visibility: visible;
+	height: 500px;
+	transition: .9s height ease-out;
+}
+
+.contact {
+	padding-bottom: 2%;
+	width: 100%;
+}
+
+.contact-form {
+	position: relative;
+	visibility: hidden;
+	background-color: #F56462; //TODO variables
+	width: 100%;
+	height: 0;
+	float: left;
+	left:0;
+	top:-66px;
+
+	//&::before {
+	//	content: '';
+	//	position: relative;
+	//	top: 0;
+	//	left: 30px;
+	//	border: transparent solid 20px;
+	//	border-right-color: #F56462; //TODO variables
+	//}
+
+	.input {
+		@include form-style;
+	}
+
+	.send {
+		@include form-style;
+		background-color: #75c398; //TODO variables
+		color: #FFFFFF; //TODO variables
```

Comment:
Please use the vars. Refer to [_vars.scss](https://github.com/peterxqe/teamfairtrade/blob/develop/src/assets/modules/_vars.scss) for correct var names.